CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0.0 - 13-03-2024
+## v1.0.1dev
+
+### `Added`
+
+### `Fixed`
+
+- [16](https://github.com/qbic-pipelines/vcftomaf/pull/16) - Template update to 3.0.2
```

Comment:
I#m not sure why, but template updates are usually not considered a mere patch release, but rather a minor version update, i.e. 1.1.0.